### DISCUSSION TITLE
CommunityToolkit/Maui Roadmap

### DISCUSSION THREAD

**Discussion by Orgbrat** [CommunityToolkit/Maui#21] [Q&A] [answered]
Do you think we will see a CommunityToolkit Maui preview when the Maui RC-1 is published. Converting a Xamarin.Forms application which uses XCT extensively to a Maui application and need to use the Community Toolkit? I know it is early but just wanted to see what I can do and what your roadmap was.

**Comment by TheCodeTraveler** [comment, marked as answer]
Yes! In fact, we are publishing the first release of `CommunityToolkit.Maui`, v1.0.0-pre1, in the next few days.

**Reply by TheCodeTraveler** [reply]
Here's a sneak peek! 
<img width="1714" alt="MauiToolkit" src="https://user-images.githubusercontent.com/13558917/131174228-52503d85-18a8-4619-b590-afacfa40be97.png">

**Comment by Orgbrat** [comment]
Thank you so much. I will be tentatively watching for the preview release...

**Reply by tcerdaj** [reply]
Hi team, there is a way to add animation parameters as Rg.Plugin.Popup does it?

I am doing the popup animation when the page is loading, but I'd like to do it when the ShowPoup method is invoked. In that way, I can use the popup for my custom toast, and other crazy design implementations.

Also when I set the position to end to the Toolkit popup is moving from top to bottom, I will like that the popup start showed from the specified position.

Thanks,

Teofilo Cerda.

**Reply by bijington** [reply]
@tcerdaj  please do not ask the same question in multiple unrelated discussions. 

Please open a new discussion.

**Comment by davefxy** [comment]
It would be great to have an AutoSuggestBox I could use on an Android. Is it possible to build on the searchhandler control to implement on?

**Comment by CliffAgius** [comment]
Closed as answered
